src/components/Modals/LiquidateSafeModal.tsx

```diff
@@ -48,15 +48,15 @@ const LiquidateSafeModal = () => {
                         })
                         popupsModel.setWaitingPayload({
                             title: 'Transaction Submitted',
-                            text: `Starting liquidation for safe #${liquidateSafePayload.safeId}...`,
+                            text: `Starting liquidation for vault #${liquidateSafePayload.safeId}...`,
                             hash: txResponse.hash,
                             status: 'loading',
                         })
-                        // txResponse.wait().then(() => {
-                        //     popupsModel.setIsWaitingModalOpen(false)
-                        //     history.push('/safes')
-                        //     closeModal()
-                        // })
+                        txResponse.wait().then(() => {
+                            closeModal()
+                            popupsModel.setIsWaitingModalOpen(false)
+                            history.go(0)
```

Comment:
Does this work when the user came from other pages? I'm not sure what 0 means

Reply:
yes, 0 just reloads the page